package.json

```diff
@@ -116,7 +118,11 @@
     "moduleNameMapper": {
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/tools/assetsTransformer.js",
       "\\.(css|less)$": "<rootDir>/tools/assetsTransformer.js"
-    }
+    },
+    "setupFiles": [
+      "raf/polyfill",
```

Comment:
Should be able to remove this by updating to the latest version of jest.  See https://github.com/facebook/jest/pull/4568

I am going to attempt it and if it is smooth I will remove this.

Reply:
This hasn't released yet. 😅 The upgrade to `jest@21` was painless though.